lib/cisco_node_utils/bgp.rb

```diff
@@ -84,7 +84,9 @@ def self.enabled
     end
 
     def self.enable(state='')
-      config_set('bgp', 'feature', state: state)
+      # IOS XR has no '[no] feature xyz'.
+      config_set('bgp', 'feature', state: state) if
+        node.client.api == 'NXAPI' || state == 'no'
```

Comment:
seems to me like the `state == 'no'` check is incorrect - you should *only* try to invoke the 'feature bgp' command on NXAPI, regardless of state